common/base.scm

```diff
@@ -167,14 +167,28 @@ The secondary value indicates the absence of an entry."
           object))))
 
 (define (replace-readtime-lookup-presented-object-or-lose string)
-  "For presentations, emacs passes something that common lisp evals at read time. The resulting object is very different than what gerbil or gambits #. tries to do, so we do everything at run time"
+  "For presentations, emacs passes something that common lisp evals at read time. The resulting object is very different than what gerbil or gambits #. tries to do, so strip the #. and pattern-match to evaluate this form at runtime."
   (let* ((pattern "#.(swank:lookup-presented-object-or-lose ")
 	 (start (string-contains string pattern)))
     (if start
         (replace-readtime-lookup-presented-object-or-lose
          (string-replace string "" start (+ 2 start)))
         string)))
 
+(define (replace-lookup-presented-object-or-lose form)
```

Comment:
As I see, it's an workaround until proper lookup is implemented, right? In which cases slime sends these commands? Perhaps some TODO-comments with context information would be helpful.